mod/wikirate/set/type/claim.rb

```diff
@@ -88,20 +88,20 @@ def edit_slot args
   def next_step_tip
     #FIXME - cardnames
     if (not topics = Card["#{card.name}+topics"]) || topics.item_names.empty?
-      "improve this claim by adding a topic."
+      "improve this note by adding a topic."
     elsif (not companies = Card["#{card.name}+company"]) || companies.item_names.empty?
-      "improve this claim by adding a company."
+      "improve this note by adding a company."
     else
       cited_in = Card.search :refer_to => card.name, :left=>{:type_id=>WikirateAnalysisID}, :right=>{:name=>Card[:wikirate_article].name}
       if card.analysis_names.size > cited_in.size
-        "cite this claim in related overviews."
+        "cite this note in related overviews."
       end
     end
   end
 
   view :sample_citation do |args|
-    tip = "easily cite this claim by pasting the following:" +
-      text_area_tag( :citable_claim, card.default_citation )
+    tip = 'easily cite this note by pasting the following:' +
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.